garage/algos/__init__.py

```diff
@@ -1,12 +1,8 @@
 from garage.algos.base import Algorithm
 from garage.algos.base import RLAlgorithm
 from garage.algos.batch_polopt import BatchPolopt
-from garage.algos.batch_polopt import BatchSampler
```

Comment:
Can you move BatchSampler into garage.sampler?

Reply:
done